src/amo/components/SecondaryHero/index.js

```diff
@@ -55,6 +66,7 @@ export const SecondaryHeroBase = ({
           <a
             className="SecondaryHero-module-link"
             href={makeCallToActionURL(module.cta.url)}
+            onClick={onHeroClick}
```

Comment:
Could this be added to `getLinkProps()` instead?

Reply:
Good idea! I can even move `href` into `getLinkProps()` too.